dash_mantine_components/figure_templates.py

```diff
@@ -0,0 +1,94 @@
+import dash_mantine_components as dmc
```

Comment:
Apparently this works, but it's a circular reference. Better to get what you need directly from `.theme`, like `from .theme import DEFAULT_THEME` as it is in `__init__.py`